core/src/main/scala/better/files/File.scala

```diff
@@ -690,7 +690,6 @@ class File private(val path: Path)(implicit val fileSystem: FileSystem = path.ge
     */
   def copyTo(destination: File, overwrite: Boolean = false)(implicit copyOptions: File.CopyOptions = File.CopyOptions(overwrite)): destination.type = {
     if (isDirectory) {//TODO: maxDepth?
-      if (overwrite) destination.delete(swallowIOExceptions = true)
```

Comment:
Hmm, this does not look right - the overwrite is not used at all in the body here? Maybe I am wrong but can you add a test for the positive case to verify behaviour i.e. `overwrite = true`? Also this method, `copyTo` should work on both files and directories (AND when destination is a file or dir - so total 4+ cases).